cfgov/v1/jinja2/v1/story_page.html

```diff
@@ -0,0 +1,36 @@
+{% extends "layout-full.html" %}
+
+{% block hero %}
+    {% for block in page.header %}
+        {% if block.block_type == 'features' %}
+            <section class="wrapper
+                            wrapper__match-content
+                            block
+                            block__bg
+                            block__border
+                            block__flush-top
+                            u-mb0">
+                {% include_block block %}
+            </section>
+        {% else %}
+            {% include_block block %}
+        {% endif %}
+    {% endfor %}
+{% endblock %}
+
+{% block content_main_modifiers -%}
+  {{ super() }} content__center-image-bleed
```

Comment:
Technically this is not correct BEM. This is creating:
```
main.content
  div.content_wrapper
    div.content_main.content__center-image-bleed
    …
```

Buuuuut the modifier `content__center-image-bleed` should be modifying `content`, not `content_main`. So the structure should be:

```
main.content.content__center-image-bleed
  div.content_wrapper
    div.content_main
    …
```

See earlier comment.

Reply:
Fixed in https://github.com/cfpb/consumerfinance.gov/commit/e9f1d87096db0a1e546dadc7b3f2dc17c0bd8d2f